### DISCUSSION TITLE
[Run issue]: Extensions

### DISCUSSION THREAD

**Discussion by ShahinGHD** [OpenTOPAS/OpenTOPAS#20] [Run-Problem] [answered]
### Contact Details

shahin.ghasemi@gmail.com

### What was the issue?

Hi everyone
Is it possible to install extensions on topas 4.0 ? 
I am assuming that there is no headers in the folder and it is not possible to install extensions at the moment, is that true?

Thanks

### TOPAS category

Geometry

### Version

4.0

### What Operative System (OS) are you running on?

Windows (WSL)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment, marked as answer]
Hi Shahin,

Yes it is possible to install extensions in OpenTOPAS 4.0.0. In principle all you need to do is navigate to the OpenTOPAS-build directory, and provide the command:

`cmake ../OpenTOPAS -DTOPAS_EXTENSIONS_DIR=/path/to/your/extensions`
`make -j10 install`

We haven't fully updated the quickStart guides to include guidelines for installing extensions yet but this is something we are working on. 

All the best,

Thongchai
